### PR TITLE
Databases week1 homework

### PR DESCRIPTION
**Pull request by vitalreddym**

### CODE REVIEW

File: databases/week1/homework.sql [hunk 1]
```diff
@@ -0,0 +1,130 @@
+
+--- task 1 ---
+SELECT
+  COUNT(id)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
_Pointer_
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+FROM
+  task;
+
+
+--- task 2  ---
+SELECT
+  COUNT(id)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+FROM
+  task
+WHERE
+  due_date IS NULL;
+
+
+--- task 3  ---
+SELECT
+  COUNT(id)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+FROM
+  task
+WHERE
+  status_id = 3;
+
+
+--- task 4  ---
+SELECT
+  COUNT(id)
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+  task.title,
+  status.name
+FROM
+  task
+  INNER JOIN status ON task.status_id = status.id;
+
+
+---- task 9 Using JOINS ----
+SELECT
+  COUNT(*),
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

File: databases/week1/homework.sql [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+  name
+FROM
+  status
+  JOIN task ON status.id = task.status_id
+GROUP BY
+  name;
```
Comment (lawansubba) [inline databases/week1/homework.sql]: status.name
Pointer
Add tablename.columnname prefix when join is involved

File: databases/week1/homework.sql [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+    status_id
+    WHEN 1 THEN 'Not started'
+    WHEN 2 THEN 'In progress'
+    WHEN 3 THEN 'Done'
+    ELSE 'Unknown'
+  END;
+  
+--- task 10  ---
+SELECT
+  COUNT(*),
```
Comment (lawansubba) [inline databases/week1/homework.sql]: COUNT(Id) as task_amount
Pointer
Add a column name to all columns created by you

**Review by lawansubba — COMMENTED** [review]
Good work! please address the few comments :)

File: databases/week1/homework.sql (hunk 5 above)
Comment (vitalreddym) [reply databases/week1/homework.sql]: ok

File: databases/week1/homework.sql (hunk 7 above)
Comment (vitalreddym) [reply databases/week1/homework.sql]: Thanks for the feedback

**Pull request merged**